strawberry/aiohttp/views.py

```diff
@@ -0,0 +1,104 @@
+import json
+from io import BytesIO
+from pathlib import Path
+
+from aiohttp import web
+from strawberry.file_uploads.data import replace_placeholders_with_files
+from strawberry.http import GraphQLHTTPResponse, process_result
+from strawberry.schema import BaseSchema
+from strawberry.types import ExecutionContext, ExecutionResult
+
+
+class GraphQLView:
+    def __init__(self, schema: BaseSchema, graphiql: bool = True):
+        self.schema = schema
+        self.graphiql = graphiql
+
+    async def __call__(self, request: web.Request) -> web.StreamResponse:
+        if request.method == "GET":
+            return await self.get(request)
+        if request.method == "POST":
+            return await self.post(request)
+        raise web.HTTPMethodNotAllowed(request.method, ["GET", "POST"])
+
+    async def get_root_value(self, request: web.Request) -> object:
```

Comment:
why object instead of Any? 😊

Reply:
@bryanculver noted in https://github.com/strawberry-graphql/strawberry/pull/895#discussion_r624566099 the following:

> Nitpick: Make these return `object` so we don't lose all type-hinting